.github/workflows/issues2md.yml

```diff
@@ -0,0 +1,38 @@
+# Action to backup issues using gh2md, contributed by
+# https://github.com/0ut0fcontrol.
+# See https://github.com/mattduck/gh2md/issues/11.
+name: Issues2Markdown
+on:
+  schedule:
+    # every day
+    - cron: "0 0 * * *"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+    - name: Backup github issues to a markdown file.
+      run: |
+        pip3 install --user --upgrade setuptools
+        pip3 install --user gh2md
+        $HOME/.local/bin/gh2md $GITHUB_REPOSITORY issues.md --idempotent
+        git add issues.md
+      env:
+        GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Commit files
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git diff --quiet && git diff --staged --quiet || git commit -am "Backup all issues into issues.md"
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+    - name: Push changes
+      uses: ad-m/github-push-action@master
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        branch: ${{ steps.extract_branch.outputs.branch }}
```

Comment:
From my understanding, this will end up attempting to push to master which won't work. We could use the artifacts upload action to host the issues.md as a pipeline artifact rather than recommitting it into the repo.

Reply:
That would work for me...but I'll have to do some research on the artifacts upload action.

Reply:
Here's an example of us using it in the build artifacts workflow. 
https://github.com/jondavid-black/AaC/blob/b3d8881f2330fd5a5713f469abcd09eab4845ac4/.github/workflows/python-build-artifacts.yml#L56-L60

I think you'll replace that last stage with something like:
```
      - name: Upload Issues Markdwon
        uses: actions/upload-artifact@v2
        with:
          name: issues.md
          path: issues.md
```

Reply:
Done!  I removed all the commit, branch, and push commands.  Now it should just do an artifact upload.